Methods to filter event data (#527)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/sdk-faqs.mdx

</div>

> ## Can I send the event data to specific destinations only?

Reviewer: ```suggestion
## Can I filter and selectively send the event data to certain destinations?
```

> Yes, you can use the [Event filtering tool](https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/event-filtering/) to specify which events should be discarded or allowed to flow through by whitelisting or blacklisting them from the UI.

Reviewer: ```suggestion
Yes, you can use RudderStack's [Client-side Event Filtering](https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/event-filtering/) feature to specify which events should be discarded or allowed to flow through - by whitelisting or blacklisting them in the RudderStack dashboard while setting up your destination.
```
